hooks/post_gen_project.py

```diff
@@ -72,177 +52,87 @@ def print_thankyou():
-
-    postgres_password = set_flag(
-        file_path,
-        "!!!POSTGRES_PASSWORD!!!",
-        value=value,
-        length=64,
-        using_digits=True,
-        using_ascii_letters=True,
-    )
-    return postgres_password
+    return set_flag(file_path, "!!!DJANGO_SECRET_KEY!!!")
 
 
-def remove_github_folder():
+def remove_graphql_files():
     file_names = [
-        os.path.join(".github"),
+        join("server/{{ cookiecutter.project_slug }}/core", "schema.py"),
+        join("server/{{ cookiecutter.project_slug }}/core", "types.py"),
+        join("server/{{ cookiecutter.project_slug }}/core", "mutations.py"),
     ]
     for file_name in file_names:
-        if os.path.exists(file_name):
-            os.remove(file_name)
+        if exists(file_name):
+            remove(file_name)
 
 
-def remove_graphql_files():
-    file_names = [
-        os.path.join("server/{{ cookiecutter.project_slug }}/core", "schema.py"),
-        os.path.join("server/{{ cookiecutter.project_slug }}/core", "types.py"),
-        os.path.join("server/{{ cookiecutter.project_slug }}/core", "mutations.py"),
-    ]
 def set_keys_in_envs():
-    env_file_path = os.path.join(".env.example")
-    postgres_init_file = os.path.join("scripts/init-db.sh")
+    env_file_path = join(".env.example")
+    postgres_init_file = join("scripts/init-db.sh")
     set_django_secret_key(env_file_path)
 
-    secret = generate_random_string(
-        length=50, using_digits=True, using_ascii_letters=True, using_punctuation=True
-    )
-    set_postgres_password(env_file_path, value=secret)
-    set_postgres_password(postgres_init_file, value=secret)
+    secret = get_random_secret_key()
+    set_flag(env_file_path, "!!!POSTGRES_PASSWORD!!!", value=secret)
+    set_flag(postgres_init_file, "!!!POSTGRES_PASSWORD!!!", value=secret)
 
-    shutil.copy2(env_file_path, os.path.join(".env"))
+    copy2(env_file_path, join(".env"))
 
 
 def main():
-
     set_keys_in_envs()
 
     if "{{ cookiecutter.client_app }}".lower() == "none":
-        shutil.rmtree("clients")
-        os.remove(os.path.join("package.json"))
-
+        rmtree("clients")
+        remove(join("package.json"))
     elif "{{ cookiecutter.client_app }}".lower() == "vue3":
-
-        remove_react_files()
+        remove_client_files("react")
         move_client_to_root("vue3")
     elif "{{ cookiecutter.client_app }}".lower() == "react":
-        remove_vue3_files()
+        remove_client_files("vue3")
         move_client_to_root("react")
 
     if "{{ cookiecutter.use_graphql }}".lower() == "n":
         remove_graphql_files()
 
-    print(INFO + "Installing necessary requirements:" + END)
-    shellscript = subprocess.Popen(
-        [
-            os.path.join(
-                "..", "{{ cookiecutter.project_slug }}", "scripts", "install-reqs.sh"
-            )
-        ],
-        stdin=subprocess.PIPE,
-    )
-    shellscript.wait()
-    shellscript.stdin.close()
-
```

Comment:
Maybe controversial, but I don't think we should be magically installing things on the user. I've moved this to be one of the INFO statements where we tell the user the script exists if they choose to use it.

Reply:
I agree. The dev will also still have to log into the CLI tools installed by install-reqs.sh, so that could be confusing if they don't know they installed them.